src/Connection/Internal/Http2Stream.php

```diff
@@ -87,12 +88,19 @@ final class Http2Stream
     /**@var Deferred|null */
     public $windowSizeIncrease;
 
+    /** @var string|null */
+    public $watcher;
+
+    /** @var int */
+    public $lastActivityAt;
+
     public function __construct(
         int $id,
         Request $request,
         Stream $stream,
         CancellationToken $cancellationToken,
         CancellationToken $originalCancellation,
+        ?string $watcher,
```

Comment:
Why is it nullable?

Reply:
If there is no inactivity timeout, no watcher is created.